protocol/testing/genesis.sh

```diff
@@ -1448,16 +1469,17 @@ function update_genesis_use_test_exchange() {
 # Modify the genesis file to add test volatile market. Market TEST-USD will be added as market 33.
 function update_genesis_use_test_volatile_market() {
 	GENESIS=$1/genesis.json
+	NEXT_MARKET_ID=33
```

Comment:
`NEXT_MARKET_ID` sounds less clear. how bout:

```suggestion
	TEST_USD_MARKET_ID=33
```